repose-aggregator/artifacts/valve/src/config/docs/DEPENDENCIES.txt

```diff
@@ -29,10 +29,9 @@ com.typesafe.akka                                   akka-actor_2.11           2.
 com.typesafe                                        config                    1.3.0            Apache License Version 2.0  https://github.com/typesafehub/config
 com.typesafe.play                                   play-json_2.11            2.5.2            Apache License Version 2.0  https://github.com/playframework/playframework
 com.typesafe.scala-logging                          scala-logging-slf4j_2.11  2.1.2            Apache License Version 2.0  https://github.com/typesafehub/scala-logging
-com.uber.jaeger                                     jaeger-apachehttpclient   0.24.0           Apache License Version 2.0  https://github.com/jaegertracing/jaeger-client-java
-com.uber.jaeger                                     jaeger-core               0.24.0           Apache License Version 2.0  https://github.com/jaegertracing/jaeger-client-java
 io.dropwizard.metrics                               metrics-core              3.2.0            Apache License Version 2.0  https://github.com/dropwizard/metrics/blob/v3.2.0/LICENSE
 io.dropwizard.metrics                               metrics-graphite          3.2.0            Apache License Version 2.0  https://github.com/dropwizard/metrics/blob/v3.2.0/LICENSE
+io.jaegertracing                                    jaeger-core               0.26.0           Apache License Version 2.0  https://github.com/jaegertracing/jaeger-client-java
```

Comment:
Looks like the version was actually bumped to `0.27`?

Reply:
Updated.